torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -157,6 +157,7 @@ class DeadCodeEliminator {
     if (it != memo_.end())
       return it->second;
     bool has_side_effects = node->kind() == prim::Print ||
+        node->kind() == aten::warn ||
```

Comment:
add this to CSE as well

Reply:
Will do in a sec! We should keep a database of those things in a single place...